### PR TITLE
Allow Multi-line Errors Underscoring

### PR DESCRIPTION
**Pull request by temanbrcom**
Rename Position into Locality and change its structure to allow using LSP4J positions directly in the language engine. Change the logic of position definition from one line-based to the range of positions

### CODE REVIEW

**Comment by temanbrcom** [comment]
@grianbrcom 
@asatklichov

**Review by grianbrcom — APPROVED** [review]
:crocodile:

**Review by asatklichov — APPROVED** [review]
👨‍🍳

File: server/src/main/java/com/ca/lsp/cobol/service/delegates/validations/CobolLanguageEngineFacade.java [hunk 1]
```diff
@@ -145,27 +141,11 @@ private static DiagnosticSeverity checkSeverity(ErrorSeverity severity) {
...
-
-    return new Range(
-        new org.eclipse.lsp4j.Position(positionLine, cPosInLn),
-        new org.eclipse.lsp4j.Position(
-            positionLine,
-            ((position.getStopPosition() - position.getStartPosition())
-                + cPosInLn
-                + ERR_POS_INDEX)));
+  private Map<String, List<Location>> convertLocations(Map<String, Collection<Location>> source) {
+    return source.entrySet().stream().collect(toMap(Map.Entry::getKey, this::repackCollection));
```
[the remark this answers is not in the document]
Comment (temanbrcom) [reply server/src/main/java/com/ca/lsp/cobol/service/delegates/validations/CobolLanguageEngineFacade.java:145]: yeah, makes sense

**Pull request merged**